poetry/publishing/uploader.py

```diff
@@ -296,7 +332,14 @@ def _upload_file(
                         f" - Uploading <c1>{file.name}</c1> <error>FAILED</>"
                     )
                 raise UploadError(e)
+            else:
+                if _ignore_existing(resp, skip_existing):
+                    if self._io.output.is_decorated():
+                        self._io.overwrite(
+                            f" - Uploading <c1>{file.name}</c1> <warning>File exists. Skipping</>"
+                        )
```

Comment:
To fix the sonarcloud issue just change the current code for this:
```suggestion
                if _ignore_existing(resp, skip_existing) and self._io.output.is_decorated():
                      self._io.overwrite(
                          f" - Uploading <c1>{file.name}</c1> <warning>File exists. Skipping</>"
                      )
```

Reply:
Hi @davidszotten , I am not sure what the policy in here is but the fix of sonarcloud it is fairly simple. 

I have added a code snipped so you can just accept the change and it should just work. 

Strange no-one from poetry looked into this, but this things happens.